cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -501,4 +501,25 @@ void setWorkflowProgramStart(Id.Program programId, String programRunId, String w
    */
   WorkflowStatistics getWorkflowStatistics(Id.Workflow workflowId, long startTime,
                                            long endTime, List<Double> percentiles);
+
+  /**
```

Comment:
The javadoc doesn't match the function signature. We are returning workflow run record.